src/base/print/printRxnFormula.m

```diff
@@ -11,14 +11,17 @@
 % INPUTS:
 %    model:             COBRA model structure
 %
-% OPTIONAL INPUTS:
+% OPTIONAL INPUTS
 %    rxnAbbrList:       Abbrs of reactions whose formulas are to be printed
+%
+% OPTIONAL INPUTS as 'ParameterName',Value pairs:
```

Comment:
Well.... We need a distinction between "optional" and Parameter/Value pairs. 
Optional is an argument with a fixed position. 
Param/Value pairs don't have a fixed position. 
And we do already have functions with Param/Value pairs, so this will need to be updated in the Documentation generation.